### DISCUSSION TITLE
ESP 8266 Wemos D1 mini (clone) Have trouble, can't connect to wifi.

### DISCUSSION THREAD

**Discussion by tpato** [revoxhere/duino-coin#1164] [Q&A]
ESP 8266 Wemos D1 mini (clone) Have trouble, can't connect to wifi. Somebody help? 
I'm using ver 2.7.5 
![Capture](https://user-images.githubusercontent.com/95827008/145291170-fb07d22e-7e40-450c-8077-731c6ea6dfe2.JPG)

![Untitled](https://user-images.githubusercontent.com/95827008/145291424-5bcefdfa-b454-448a-b435-caa4d0da46f6.png)

**Comment by revoxhere** [comment]
Make sure the wifi is 2.4 GHz and the SSID and Password are correct
